documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -193,20 +193,15 @@ kubectl apply -f _kafka.yaml_
 +
 If you include the Topic Operator in the deployment, the Topic Operator will delete all the topics.
 
-. Run an `exec` command to one of the Kafka broker pods to open the ZooKeeper shell script.
+. Delete the internal topic store topics from the Kafka cluster.
 +
 For example, where _my-cluster-kafka-0_ is the name of the broker pod:
 +
-[source,shell,subs="+quotes,attributes"]
-----
-kubectl exec -ti _my-cluster-zookeeper-0_ -- bin/zookeeper-shell.sh localhost:12181
-----
-. Delete the whole `/strimzi` path to remove the Topic Operator storage:
-+
-[source,shell,subs="+quotes,attributes"]
+[source,shell]
 ----
-deleteall /strimzi
+kubectl exec -ti my-cluster-kafka-0 -- ./bin/kafka-topics.sh --bootstrap-server localhost:9092 --topic __strimzi-topic-operator-kstreams-topic-store-changelog --delete && ./bin/kafka-topics.sh --bootstrap-server localhost:9092 --topic __strimzi_store_topic --delete
```

Comment:
We should use the following:

```
kubectl -n kafka run kafka-admin -ti --image={DockerKafkaImageCurrent} --rm=true --restart=Never -- ./bin/kafka-topics.sh --bootstrap-server localhost:9092 --topic __strimzi-topic-operator-kstreams-topic-store-changelog --delete && ./bin/kafka-topics.sh --bootstrap-server localhost:9092 --topic __strimzi_store_topic --delete
```

We should probably also add some caveat that when needed the command has to be adapted to correspond to user's configuration of listeners and authentication.

Reply:
Changed to:

> `kubectl -n kafka run kafka-admin -ti --image={DockerKafkaImageCurrent} --rm=true --restart=Never -- ./bin/kafka-topics.sh --bootstrap-server localhost:9092 --topic __strimzi-topic-operator-kstreams-topic-store-changelog --delete && ./bin/kafka-topics.sh --bootstrap-server localhost:9092 --topic __strimzi_store_topic --delete`
> 
> The command must correspond to the type of listener and authentication used to access the Kafka cluster.